Flepper.Tests.Unit/QueryBuilder/Commands/OrderByCommandTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using Flepper.QueryBuilder;
+using FluentAssertions;
+using Xunit;
+
+namespace Flepper.Tests.Unit.QueryBuilder.Commands
+{
+    [Collection("CommandTests")]
+    public class OrderByCommandTests
+    {
+        [Fact]
+        public void ShouldCreateSelectWithOrderByStatement()
+        {
+             FlepperQueryBuilder.Select("Id", "Name", "Birthday")
+                .From("user")         
+                .OrderBy("Name")
+                .Build()
+                .Trim()
+                .Should()
+                .Be("SELECT [Id],[Name],[Birthday] FROM [user] ORDER BY [Name]");
+        }
+
+        [Fact]
+        public void ShouldCreateSelectWithOrderByDescStatement()
+        {
+            FlepperQueryBuilder.Select("Id", "Name", "Birthday")
+               .From("user")
+               .OrderByDesc("Name", "Birthday")
+               .Build()
+               .Trim()
+               .Should()
+               .Be("SELECT [Id],[Name],[Birthday] FROM [user] ORDER BY [Name],[Birthday] DESC");
+        }
+        //SELECT [Id],[Name],[Birthday] FROM [user] ORDER BY [Name],[Birthday] DESC
+        //SELECT [Id],[Name],[Birthday] FROM [user] ORDER BY [Name] DESC
+        [Fact]
+        public void ShouldCreateSelectWithWhereAndOrderByStatement()
+        {
+            var queryResult = FlepperQueryBuilder.Select("Id", "Name", "Birthday")
+                .From("user")
+                .Where("Name").EqualTo("Fabio")
+                .OrderBy("Name")
+                .BuildWithParameters();
+
+            queryResult
+                .Query
+                .Trim()
+                .Should()
+                .Be("SELECT [Id],[Name],[Birthday] FROM [user] WHERE [Name] = @p0 ORDER BY [Name]");
+
+            dynamic parameters = queryResult.Parameters;
+
+            Assert.Equal("Fabio", parameters.@p0);
+        }
+
+        [Fact]
+        public void ShouldCreateSelectWithWhereAndOrderByDescStatement()
+        {
+            var queryResult = FlepperQueryBuilder.Select("Id", "Name", "Age")
+                .From("user")
+                .Where("Age").EqualTo(29)
+                .OrderByDesc("Name")
+                .BuildWithParameters();
+
+            queryResult
+                .Query
+                .Trim()
+                .Should()
+                .Be("SELECT [Id],[Name],[Age] FROM [user] WHERE [Age] = @p0 ORDER BY [Name] DESC");
+
+            dynamic parameters = queryResult.Parameters;
+
+            Assert.Equal(29, parameters.@p0);
+        }
+
+        [Fact]
+        public void ShouldCreateSelectWithWhereAndInnerJoinOrderByStatement()
+        {
+            var queryResult = FlepperQueryBuilder.Select("Id", "Name", "Age")
```

Comment:
We have a problem when executing this query because the order does not specify which table belongs to this column.

Should be something like that: 

```
var queryResult = FlepperQueryBuilder.Select("Id", "Name", "Age")
                  .From("user").As("t1")
                  .InnerJoin("address").As("t2").On("t2", "column1").EqualTo("t1", "column2")
                  .Where("Age").EqualTo(29)
                  .OrderBy("t1","Name")
```

Reply:
Nice!

Reply:
You're right. I'll think of a solution for this soon